en/frames_multicopter/holybro_x500_pixhawk4.md

```diff
@@ -0,0 +1,198 @@
+Unscrew the landing gear screws and insert the vertical pole, see figures 1 and 2.
+
+**Step 2:** We proceed to arm the motor holder by using 4 U-shaped nylon straps to attach the holder to the carbon fiber arm as shown in figure 3.
+
+**Step 3:** Attach the power management PM02 to the bottom plate as shown in Figures 4 and 5.
+
+**Step 4:**  Assemble the lower plate to the landing gears.
+Screw the landing gear with a vertical pole to the bottom plate. The lower plate has 4 holes (see Figure 4 arrows) use the M3X8 screws, a total of 8 pieces, 4 on each side. 
+
+**Step 5:** Assembling the Battery Mount to the frame. 
+For this we will need the M2 5X6 screws and the battery mount see Figure 7. Insert the long rods to the small rings see Figure 8 and 9. Also, with the battery holder completely armed, screw it where arrow shown in the image as shown in Figure 10; keep in mind GPS module will be facing front.
+
+**Step 6:** Assemble the 8*3 2.54mm pitch Horizontal Pin to the 10 to 10 pin cable (PWM) to the Power Management Board. 
+Connect the 10 to 10 pin cable (PWM) to the 8*3 2.54mm pitch Horizontal Pin, see Figure 11. Cut a piece of 3M Tape and attach to the bottom of the Horizontal Pin, see Figure 12  stick the Horizontal Pin to the Power Management Board, see Figure 13. 
+
+**Step 7:** Install vehicle Arms to the main body.
+Take the ESC and push it in the Arm tube as shown in FIgure 14 and make sure that the ESC cables are not too long since we will push them back in as shown in Figure 15.
+
+**Step 8:** Assemble arm to main body.
+With 4 more U-shaped nylon straps attach the arm with the motor installed to the body of the vehicle as shown in Figure 16, this way you also attach the bottom plate to the top plate. Also keep in mind to have the arm tube a bit pushed in to that it can be kept securely in place as shown in the Red Square from Figure 16.
+
+**Step 9:** Connect Motors cables.
+After the 4 arms are mounted on to the main body, connect the cables (red, blue, black) and push them into the arm tube, see Figures 17. The 3 cables that are color-coded go connected to the ESC. 
+
+**Step 10:** Mounting the GPS on the frame.
+For this, we will need the Pixhawk 4 GPS and the mounting plate. Mount GPS mast to the plate, use the 4 screws see the red circle in Figure 18, keep in mind that the plate is mounted to the battery holder tubes as indicated by the arrows in Figure 18. Use the tape and stick the GPS to the top of the GPS mast, see Figure 19.
+
+**Step 11:** Pixhawk 4 wiring
+The Pixhawk 4, which has several different wires and connections with it. Included below is a picture of every wire needed with the Pixhawk and how it looks when connected.
+Plugin Telemetry and GPS module to the flight controller as seen in Figure 20; plug in the RC receiver, all 4 ESCs to the flight controller as well as the power module as shown in Figure 21.
+
+
+### Vehicle calibration/setup
+Vehicle calibration/setup is similar for all vehicles. 
```

Comment:
@Junkim3DR What airframe should be used? Can you update the image in https://github.com/PX4/px4_user_guide/pull/681 - number 30 because it is unreadable. Feel free to use full size resolution - I can shrink them if needed.

Reply:
@hamishwillee New commit submitted, images renamed, folder generated and reuploaded both image 30.